### ISSUE TITLE
More Dataset Information

### ISSUE THREAD

**Issue by Tonthatj** [nyukat/breast_cancer_classifier#19]
Hi All, 

I am currently testing out your suite of models and was hoping to learn more about the dataset on which the models were trained. I have read your paper describing the data in detail, but I could not find the answer to my question in it. I was curious as to what is the statistical distribution of the original dicom images in regards to the Relative X-Ray Exposure as well as the Exposure Index/Sensitivity values found in the dicom tags?

**Comment by kjgeras** [comment]
Hi Tonthatj,

We didn't extract these from the dicom files. How do you think this could be useful?

**Comment by Tonthatj** [comment]
I have noticed after reviewing my own data set that your model performs better or worse on different subsets based on different dicom attributes. For example any mammogram with a relative xray exposure > 10000 we noticed a significant drop in the models auc, roughly 30%. The same could be said if the magnification was different from 1. The model also performed variably for the different sensitivity levels/exposure indexes. Would it be possible for you to generate the statistics for the various dicom tags?
